### PR TITLE
ENT - RMP-331 "Invalid validation message for autocomplete"

### PR DESCRIPTION
**Pull request by gayan-orangehrm**
Vue App changes related to this PR - https://websvn.orangehrm.com/orangehrm?op=comp&compare[]=/enterprise/branch/recruitment-vue-conversion/@171290&compare[]=/enterprise/branch/recruitment-vue-conversion/@171291

Apart from AutocompleteInput.vue and autocomplete-input.scss all the other files contains auto applied lint fixes

### CODE REVIEW

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue [hunk 1]
```diff
@@ -220,7 +230,11 @@ export default defineComponent({
         this.loading = false;
         this.dropdownOpen = false;
         this.pointer = -1;
-        this.$emit('update:modelValue', null);
+        this.searchTerm = null;
+        this.$emit(
+          'update:modelValue',
+          this.multiple ? this.selectedValues : this.searchTerm,
```
Comment (lgtkaushalya) [inline components/src/core/components/Input/Autocomplete/AutocompleteInput.vue]: I think this is incorrect. For multi-select option, it'll always return this.selectedValues

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue [hunk 2]
```diff
@@ -129,10 +128,14 @@ export default defineComponent({
     dropdownPosition: {
       type: String,
       default: BOTTOM,
-      validator: function(value: Position) {
+      validator: function (value: Position) {
         return DROPDOWN_POSITIONS.indexOf(value) !== -1;
       },
     },
+    isQuickSearch: {
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue]: Already removed

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue (hunk 1 above)
Comment (gayan-orangehrm) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue]: this has been removed "this.selectedValues"

File: components/src/core/components/Input/Autocomplete/AutocompleteInput.vue [hunk 3]
```diff
@@ -268,16 +284,31 @@ export default defineComponent({
       this.dropdownOpen = false;
       this.pointer = -1;
       this.$emit('dropdown:blur');
+      if (!this.selectedValues && !this.isQuickSearch) {
```
[the remark this answers is not in the document]
Comment (gayan-orangehrm) [reply components/src/core/components/Input/Autocomplete/AutocompleteInput.vue]: "isQuickSearch" is removed and no longer available

**Pull request merged**